srcpkgs/hstr/template

```diff
@@ -1,22 +1,19 @@
 # Template file for 'hstr'
 pkgname=hstr
-version=2.3
+version=2.5
 revision=1
 build_style=gnu-configure
-hostmakedepends="automake pkg-config"
+hostmakedepends="autoconf automake pkg-config"
```

Comment:
autoconf is already a dependency of automake

Reply:
fixed!